### PR TITLE
Allow newer numpy versions

### PR DESCRIPTION
**Pull request by daniel-wer**
### Description:
- This will allow to use newer numpy versions in tools that use this dependency, for example, voxelytics
- The webknossos package also uses version 1.21.1, currently

### CODE REVIEW

File: wkcuber/wkcuber/convert_nifti.py [hunk 1]
```diff
@@ -196,7 +196,6 @@ def convert_nifti(
     # Writing wkw compressed requires files of shape (shard_size, shard_size, shard_size)
     # Pad data accordingly
     padding_offset = shard_size - np.array(cube_data.shape[1:4]) % shard_size
-    padding_offset = (0, 0, 0)
```
Comment (daniel-wer) [inline wkcuber/wkcuber/convert_nifti.py:199]: No idea what this what supposed to accomplish :shrug: Looks like an oversight after a refactoring to me (and mypy complained) which is why I removed it.

Comment (daniel-wer) [reply wkcuber/wkcuber/convert_nifti.py:199]: @philippotto It seems like you wrote this code. Do you remember whether you deactivated the padding deliberately?

**Review by jstriebel — APPROVED** [review]
LGTM :+1: (except for the `padding_offset = (0, 0, 0)` part, let's see)

File: wkcuber/wkcuber/convert_nifti.py (hunk 1 above)
Comment (philippotto) [reply wkcuber/wkcuber/convert_nifti.py:199]: I think this line simply slipped through while refactoring. Removing it should be fine in my opinion :+1:

**Pull request merged**